doc/profiling.rst

```diff
@@ -220,9 +220,16 @@ As an example::
 Interface to Third Party Profiling Tools 
```

Comment:
Please also update the line in this file that says that "PSyclone currently supports dl_timer"